### DISCUSSION TITLE
Suddenly no css of deployed website

### DISCUSSION THREAD

**Discussion by kxzo** [vercel/community#2015] [Help]
### Summary

Hey, so I started developing a landing page for my project that has had over 200 deployments on vercel rn. before everything worked fine, so did the css - I use Tailwindcss. But now after installing react-parallax, creating my index page and even after removing the index page and replacing it with my old one - that used to work, it doesn't work anymore. You can visit my page at: [Nympt](https://nympt.vercel.app/de). I am very frustrated cause everything worked and now I don't have any styles anymore. pretty weird. What could I have done wrong?

On localhost and when building locally, **everything works fine**!

This is the current console (there are some errors but Idk why)

![grafik](https://user-images.githubusercontent.com/66639485/230729236-e34c2a1f-6480-4e6b-815d-f01625c41e78.png)

### Additional information

On localhost and when building locally, **everything works fine**

### Example

https://nympt.vercel.app/de

**Comment by amyegan** [comment]
Thanks for sharing that example link, @einKnuffy. It looks like your CSS is loading when I visit the page. Did you already find a solution?
